### ISSUE TITLE
Watermark doesn't show up in generated pdf

### ISSUE THREAD

**Issue by mubblemanish** [bpampuch/pdfmake#476]
Hello

I set the document definition as 
dd = {
  watermark: 'Watermark Text',
  content: [{text: 'Some random text'}]
}

The generated pdf had no watermark in the page.

**Comment by vbuser2004** [comment]
Possible answer here:

http://stackoverflow.com/a/32658671/1972353

Good luck!

**Comment by mubblemanish** [comment]
@vbuser2004 Thanks for the reply; yesterday I also saw this post and eventually figured out too. Apart from this I made few more changes to make watermark work as I wanted. Below are the changes I did
in printer.js

pdfKitDoc.fill('darkgray');                 // Change colour
pdfKitDoc.opacity(0.3);                   // Change opacity as watermark was too dark for my liking

Commented out below to keep the watermark text horizontal

//var angle = Math.atan2(pdfKitDoc.page.height, pdfKitDoc.page.width) * 180/Math.PI;
//pdfKitDoc.rotate(0, {origin: [pdfKitDoc.page.width/2, pdfKitDoc.page.height/2]});

// Changed width and height for proper rendering of watermark text horizontally

pdfKitDoc.addContent('' + (pdfKitDoc.page.width/2 - watermark.size.size.width/4) + ' ' + (pdfKitDoc.page.height/2 - watermark.size.size.height/4) + ' Td');

Ideally, colour, opacity & text orientation should be configurable.
